packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -1160,6 +1167,52 @@ QUnit.module('aria accessibility', {
         assert.equal($item0.attr('aria-hidden'), 'true', 'aria-hidden is true for 1st item');
         assert.equal($item1.attr('aria-hidden'), undefined, 'aria-hidden does not exist for 2nd item');
     });
+
+    QUnit.test('element should contain correct aria attributes', function(assert) {
+        const $element = $('#multiView').dxMultiView({ items: [1, 2] });
+
+        const attributeMap = {
+            'role': 'group',
+            'aria-roledescription': 'MultiView',
+            'aria-label': 'Use the arrow keys or swipe to navigate between views',
+        };
+
+        Object.keys(attributeMap).forEach(key => {
+            assert.strictEqual($element.attr(key), attributeMap[key], `${key} attribute is correct`);
+        });
+    });
+
+    function checkAttributes(assert, items) {
```

Comment:
it's more intuitive if functions are declared in the beginning of a scope

Reply:
[e88d35eb9fa4d0fcb171faaf710e3d6ef8dba941]